lib/react_client/react_interop.dart

```diff
@@ -423,8 +425,8 @@ class ReactDartInteropStatics2 implements ReactDartInteropStatics {
 /// See [ReactDartInteropStatics], [createReactDartComponentClass].
 class ComponentStatics<T extends Component> {
   final ComponentFactory<T> componentFactory;
-
-  ComponentStatics(this.componentFactory);
+  final Component2 componentInstance;
+  ComponentStatics(this.componentFactory, {this.componentInstance});
```

Comment:
What is the reason that a `componentInstance` must be stored in here now?

Reply:
Will be addressed in a doc comment.